shared/Microsoft.Extensions.ActivatorUtilities.Sources/sharedsources.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <ItemGroup>
+    <Compile Include="$(NuspecBasePath)\..\Microsoft.Extensions.ParameterDefaultValue.Sources\ParameterDefaultValue.cs">
```

Comment:
Can a source package reference another source package? Could we potentially leave `ParameterDefaultValue` in common and use a package reference?